android/src/main/cpp/NativeProxy.cpp

```diff
@@ -348,19 +349,41 @@ void NativeProxy::installJSIBindings(
 
   std::weak_ptr<ErrorHandler> weakErrorHandler = errorHandler;
 
+  auto convertToArray = [](jsi::Runtime &rt, const std::string &value) {
+    std::vector<float> transformMatrixList;
+    std::istringstream stringStream(value);
+    std::copy(
+        std::istream_iterator<float>(stringStream),
+        std::istream_iterator<float>(),
+        std::back_inserter(transformMatrixList));
+    assert(transformMatrixList.size() == 9);
+    jsi::Array matrix(rt, 9);
+    for (int i = 0; i < 9; i++) {
+      matrix.setValueAtIndex(rt, i, transformMatrixList[i]);
+    }
+    return matrix;
+  };
+
   layoutAnimations->cthis()->setAnimationStartingBlock(
-      [wrt, weakModule, weakErrorHandler](
+      [wrt, weakModule, weakErrorHandler, convertToArray](
```

Comment:
Can we convert `convertToArray` from lambda to a function (outside of `NativeProxy.cpp`, maybe some utils)?

Reply:
:+1: for defining it as a separate function.